### PR TITLE
feature: added product listing page

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added product listing page
Preview: https://deploy-preview-5--practical-feynman-c4afb1.netlify.app/pages/store/store.html

### CODE REVIEW

**Review by amarphule — COMMENTED** [review]
Looks good. one suggestion make card height equal so it will look nice
om-card

File: pages/store/store.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+}
+.filter-container input {
+  width: 1rem;
+  height: 1rem;
+  margin-right: 8px;
+}
+.store-heading-container {
+  font-size: 1rem;
+}
+#price-range-slider {
```
Comment (raghav2110) [inline pages/store/store.css:62]: Any specific reason for using id here ?

File: pages/store/store.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  grid-template-areas: "filter store store store store";
+}
+
+.store-container {
+  grid-area: store;
+  margin: 1rem 0 0 1rem;
+}
+.store-vertical-products {
```
Comment (raghav2110) [inline pages/store/store.css:11]: Naming of class can be more better.

**Review by raghav2110 — APPROVED** [review]
Overall the code looks good. I put few comments what I thought can be improved apart from that code looks great. Also website is amazing.

**Comment by omgaikwad** [comment]
> Looks good. one suggestion make card height equal so it will look nice om-card

Sure!

File: pages/store/store.css (hunk 2 above)
Comment (omgaikwad) [reply pages/store/store.css:11]: Yes

File: pages/store/store.css (hunk 1 above)
Comment (omgaikwad) [reply pages/store/store.css:62]: Actually yes there is a reason when I use class the slider doesn't work but everything work perfectly when I use ID

**Pull request merged**